src/sbvr-api/control-flow.ts

```diff
@@ -64,3 +69,32 @@ export const getMappingFn = (headers?: {
 		return mapTill;
 	}
 };
+
+export const delay = (ms: number) =>
+	new Promise<void>((resolve) => setTimeout(resolve, ms));
+
+export const fromCallback = <T>(
+	resolver: (callback: (err: any, result?: T) => void) => void,
+): Promise<T> =>
+	new Promise<T>((resolve, reject) => {
+		resolver((err, result?: T) => {
+			if (err || !result) {
```

Comment:
I didn't expect to have `|| !result` here :thinking: 
Aren't falsy results valid callback result types?

Reply:
You're right, I'll stop taking shortcuts on the typing

Reply:
Nevermind, I can't type it quite right because it has to be compatible with the places it's passed to which also take shortcuts so I've gone with a cast instead